### PR TITLE
remove controllers that have been replaced by django views

### PR DESCRIPTION
**Pull request by asmacdo**
Removes webpy controllers that have been replaced by django views.

### CODE REVIEW

File: server/test/unit/test_plugin_manager.py [hunk 1]
```diff
@@ -73,6 +73,7 @@ def test_types(self):
         """
 
         # Setup
+        types_db.clean()
```
Comment (beav) [inline server/test/unit/test_plugin_manager.py]: Is this change intentional?

Comment (asmacdo) [inline server/test/unit/test_plugin_manager.py]: Yes, I should have mentioned that. One of the tests that I deleted cleans up after one of the tests that remains. When I removed the webpy tests, it caused this test to fail because it was operating on an unclean database.

**Comment by beav** [comment]
This looks ok to me but someone with more familiarity with the Django work should probably review as well.

**Comment by bmbouter** [comment]
@asmacdo there are some flake8 issues with these. would you run flake8 on all the files and push a fix for those?

**Comment by bmbouter** [comment]
Besides the flake8 LGTM

**Comment by bmbouter** [comment]
@asmacdo also can you track this issue here. https://pulp.plan.io/issues/148
